src/main/java/me/totalfreedom/totalfreedommod/bridge/CoreProtectBridge.java

```diff
@@ -1,11 +1,7 @@
 package me.totalfreedom.totalfreedommod.bridge;
 
 import java.io.File;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
```

Comment:
What was the reason for changing this? Fairly sure wildcard imports are bad practice?

Reply:
well yeah, but this looks good there shouldn't be any name conflict

Reply:
Using wildcard imports will just make the compilation a little bit longer as it will need to search through the dependency and get what is needed.

Reply:
This was done by intellij automatically, whenever multiple imports are selected. I can fix this but the difference is negligible